dist/webpack/lazy-controller-loader.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+'use strict';
+
+var generateLazyController = require('./generate-lazy-controller');
+
+var getStimulusCommentOptions = require('../util/get-stimulus-comment-options');
+/**
+ * Loader that can make a Stimulus controller lazy.
+ *
+ * This loader is meant to be used to load the Stimulus controllers
+ * themselves. It detects a stimulusLazyController: true comment above the
+ * controller. If present, the controller is replaced by a controller
+ * that will lazily import the real controller the first time the
+ * element appears.
+ *
+ * @param {String} source of a module that exports a Stimulus controller
+ * @return {String}
+ */
+
+
+module.exports = function (source) {
```

Comment:
where is the `src/` file corresponding to this `dist` file ?

Reply:
Thx for catching - this was added by accident, it will be in another PR.